ngs_mapper/nfilter.py

```diff
@@ -104,14 +104,25 @@ def is_valid(fn):
     msg= "Skipped files %s that were not within chosen platforms %s" % ( plat_files, platforms)
     if not files:
         raise ValueError("No fastq or sff files found in directory %s" % readsdir + '\n' + msg)
-    if parallel:
+    if threads > 1:
```

Comment:
This looks like it would work fine, but I'm wondering why you took this approach instead of just `pool = multiprocessing.Pool(threads)`